### ISSUE TITLE
When a new conversation is started, it is not appearing in the chats list

### ISSUE THREAD

**Issue by marcosnc08** [cometchat-pro/cometchat-chat-uikit-react#134]
### Describe the problem

> I am sending a message to a user using the api. After sending the message I go to the chat and I don't see the new chat with the sent message. If I reload the chat, I can see the message sent.

### What was the expected behavior?

> Receive the new message in real-time

### Reproduction

- Integrate the UI Kit version 4
- Send a message to the user using the API

### Environment

> Please provide the following:

- React UI Kit v4

I would like to know if I am missing something in the docs, or should implement this myself somehow.

Thanks!

**Comment by marcosnc08** [comment]
Hey!

This is de CURL request I am making to the API:

`curl --request POST \
  --url https://246369726b4a0e47.api-us.cometchat.io/v3/messages \
  --header 'Content-Type: application/json' \
  --header 'User-Agent: insomnia/8.2.0' \
  --header 'accept: application/json' \
  --header 'apikey: 3f15fd58037fcb4adfceb26d3e4d7c9f6c2751ac' \
  --header 'content-type: application/json' \
  --header 'onBehalfOf: superhero2' \
  --data '{
	"category": "message",
	"type": "text",
	"data": {
		"text": "Hi Tom!"
	},
	"receiver": "superhero1",
	"receiverType": "user"
}'`

Basically when having the chat open logged in with superhero1, and, is the first message from superhero2, the message does not appear on the list, but I can hear the sound for a new message.

After fully reloading the page, I see the message.
